plugin/trino-geospatial/src/test/java/io/trino/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -1494,22 +1494,19 @@ public void testGeometryJsonConversion()
         assertValidGeometryJson(
                 "{\"type\":\"MultiLineString\", \"coordinates\":[[[0.0,0.0],[1,10]],[[10,10],[20,30]],[[123,123],[456,789]]]}",
                 "MULTILINESTRING ((0 0, 1 10), (10 10, 20 30), (123 123, 456 789))");
+        assertValidGeometryJson("{\"type\":\"FeatureCollection\",\"features\":[]}", "GEOMETRYCOLLECTION EMPTY");
+
+        // Valid JSON with invalid Geometry definition but return empty
```

Comment:
This sounds regression. Can you keep the original behavior?

Reply:
I followed their behavior in 1.19.0, which you can see here.

https://github.com/locationtech/jts/blob/1.19.0/modules/io/common/src/test/java/org/locationtech/jts/io/geojson/GeoJsonReaderTest.java#L72

Reply:
Should we just use `1.18.2` since `1.19.0` introduces [add support for GeoJSON's Feature and FeatureCollection types](https://github.com/locationtech/jts/pull/837)?

Reply:
No, I would recommend using the latest version.

Reply:
Should I resolve this conversation since using the latest version with support for FeatureCollection?

Reply:
I need some assistance in resolving this.

Should I change the tests since I'm using `1.19.0`?
Thanks.

Reply:
Can you confirm if new library's behavior follows [RFC 7946](https://tools.ietf.org/html/rfc7946)?

Reply:
Can you confirm if new library's behavior follows [RFC 7946](https://tools.ietf.org/html/rfc7946)?

Reply:
I'm working on it.

Reply:
From what I can see, it should follow the RPC7946.

1. A GeoJSON object with the type "FeatureCollection" is a FeatureCollection object.
2. A FeatureCollection object has a member with the name "features."
3. The value of "features" is a JSON array.